Twig/NodeExpressionExtensionReference.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * Copied from twigphp/twig.
+ *
+ * (c) 2009 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace JMS\TranslationBundle\Twig;
+
+/**
+ * Represents an extension call node.
+ * Deprecated since twig 1.23 and removed in 2.0.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ */
+class NodeExpressionExtensionReference extends \Twig_Node_Expression
```

Comment:
Thank you. Im not too sure what to do there to be honest. Im also not sure what the purpose of `\Twig_Node_Expression_ExtensionReference` is. 

We do not need the new `NodeExpressionExtensionReference` for Twig1 because `\Twig_Node_Expression_ExtensionReference` exists. And we should be able to find a way around this for Twig2, without copying files from Twig1.